applications_tests/gls_navier_stokes_2d/mms2d_carreau_1st-order_gls.prm

```diff
@@ -13,18 +13,17 @@ end
 # Physical Properties
 #---------------------------------------------------
 subsection physical properties
-set number of fluids = 1
```

Comment:
Do you want to remove it since the default value is 1?

Reply:
yeah i'm not sure it's worth specifying if the default value is good. I prefer the "don't rewrite the parameter if the default value is used approach" since it forces us to have good and meaningful default values :)